### DISCUSSION TITLE
how do i uninstall windows subsystem for android [ wsa ] with magisk root and google play store

### DISCUSSION THREAD

**Discussion by myusrn** [LSPosed/MagiskOnWSALocal#645] [Q&A]
How do i uninstall windows subsystem for android [ wsa ] with magisk root and google play store installed in the C:\wsaMagiskGapps directory?

**Comment by Almighty188** [comment]
Right Click "Start Menu" in Windows, then choose "Installed apps", then look for "Windows Subsystem for Android™" , click the three dots on  the right hand side and select "Uninstall"

**Comment by myusrn** [comment]
Thanks @Almighty188 for those details. Curious if anyone has found wsa with magisk and playstore install will initialize unexpectedly when you click a URL link in windows desktop email. That's started happening on my setup recently which is what had me considering doing an uninstall. Also has anyone found having it installed can lead to what seems like dune kind of background processing that causes enough CPU load to spin up and keep laptop fans running.

**Reply by Almighty188** [reply]
You are very welcome, @myusrn.  I had something similar happening and it seems to always happened, basically the Windows Facebook Messenger app when clicking on links is opening the Facebook app in WSA instead of opening Facebook in the browser.   The laptop issue is more of bad thermal design that causes the CPU to thermal throttle.
